src/components/project/form-row.vue

```diff
@@ -16,10 +16,10 @@ except according to the terms contained in the LICENSE file.
     </td>
     <td class="form-name">
       <template v-if="canLinkToFormOverview">
-        <router-link :to="primaryFormPath(form)">{{ form.nameOrId }}</router-link>
+        <form-link :form="form"/>
       </template>
       <template v-else-if="canLinkToSubmissions">
-        <router-link :to="submissionsPath.all">{{ form.nameOrId }}</router-link>
+        <form-link :form="form" :to="submissionsPath.all"/>
```

Comment:
`canRoute()` and `LinkIfCan` don't always work that well on the homepage, which is why `<router-link>` + `v-if` was used here. For example, form pages will check the `form` resource, but the `form` resource isn't set on the homepage, so it will never fail `validateData` when checked from the homepage.

Here, the default value of the `to` prop is `primaryFormPath()`. However, given the behavior of `canRoute()` on the homepage, `primaryFormPath()` won't ever return a path to the submissions page. That's why we specify `submissionsPath.all` explicitly.